.github/workflows/backend-prod-ci.yml

```
@@ -52,3 +52,17 @@ jobs:
 
       - name: Build with Gradle
         run: ./gradlew build -x test
+
+      - name: Upload app.jar to S3
+        run: |
+          FULL_SHA=${{ github.sha }}
+          SHORT_SHA=$(echo $FULL_SHA | cut -c1-7)
+          BUILD_TIME=$(TZ=Asia/Seoul date +'%Y-%m-%d_%H-%M-%S')
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)